examples/2d/move_sprite.rs

```diff
@@ -0,0 +1,44 @@
+use bevy::prelude::*;
+
+fn main() {
+    App::build()
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup.system())
+        .add_system(sprite_movement.system())
+        .run();
+}
+
+enum Direction {
+    Up,
+    Down,
+}
+
+fn setup(
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+    mut materials: ResMut<Assets<ColorMaterial>>,
+) {
+    let texture_handle = asset_server.load("branding/icon.png");
+    commands.spawn_bundle(OrthographicCameraBundle::new_2d());
+    commands
+        .spawn_bundle(SpriteBundle {
+            material: materials.add(texture_handle.into()),
+            ..Default::default()
+        })
```

Comment:
could you add a transform component to show how to place a sprite from the start?